taxi/forms.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, get_user_model
+from django.core.exceptions import ValidationError
+
+from .models import Car
+
+
+class DriverCreationForm(UserCreationForm):
+    first_name = forms.CharField()
+    last_name = forms.CharField()
+    email = forms.EmailField()
+
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "username",
+            "email", "password1", "password2",
+            "license_number",
```

Comment:
The field `license_number` is included in the `fields` tuple, but it is not defined in the `DriverCreationForm`. Ensure that the `license_number` field is defined in the form or remove it from the `fields` tuple.